tests/main/security-dev-input-event-denied/task.yaml

```diff
@@ -54,16 +54,15 @@ execute: |
 
     echo "When the joystick plug is connected"
     snap connect test-snapd-event:joystick
+    udevadm trigger
     udevadm settle
 
     # Note, '-event-kbd' devices aren't joysticks (those are -event-joystick
     # (evdev event*) and -joystick (js*)) and therefore shouldn't be added to
     # the device cgroup when the joystick interface is plugged.
     echo "Then the snap is still not able to access an evdev keyboard"
-    if test-snapd-event "-event-kbd" 2> call.error; then
-        echo "Expected permission error calling evtest with connected joystick plug"
-        exit 1
-    fi
+    retry -n 3 --wait 1 sh -c '! test-snapd-event "-event-kbd" 2> call.error'
```

Comment:
I wonder if we shouldn't bump this, 3 seconds total seems a bit low?

Reply:
I ran hundreds of times with 3 seconds and 0 fails, not sure if more retries are needed.